templates/run_neo4j.java.qute

```diff
@@ -0,0 +1,31 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+//JAVA 10+
+//DEPS org.neo4j.driver:neo4j-java-driver:4.3.4
+
+// NEO4J_URL="bolt://localhost" NEO4J_PASSWORD="secret"  ./{baseName}.java "MATCH (n) RETURN labels(n) AS labels, properties(n) AS props LIMIT 10"
+import static java.lang.System.*;
```

Comment:
Not a fan of static system imports, actually, I think they are terrible.

Reply:
Copied from the sample I was sent, I can revert that